tests/clar_libgit2.c

```diff
@@ -558,5 +558,36 @@ bool cl_sandbox_supports_8dot3(void)
 
 	return supported;
 }
+
+bool cl_sandbox_supports_reparse_points(void)
+{
+	unsigned int flags = 0;
+	char path[8];
+	char drive = 'A' + _getdrive() - 1;
+
+	sprintf(path, "%c:\\", drive);
+
+	GetVolumeInformation(path, NULL, 0, NULL, NULL, &flags, NULL, 0);
+
+	return flags & FILE_SUPPORTS_REPARSE_POINTS;
+}
 #endif
 
+bool cl_sandbox_supports_links(void)
+{
+#ifdef GIT_WIN32
+	unsigned int path_flags = 0;
+	unsigned int cl_flags = 0;
+	char path[8];
+	char drive = 'A' + _getdrive() - 1;
+
+	sprintf(path, "%c:\\", drive);
+
+	GetVolumeInformation(_cl_sandbox, NULL, 0, NULL, NULL, &cl_flags, NULL, 0);
+	GetVolumeInformation(path, NULL, 0, NULL, NULL, &path_flags, NULL, 0);
```

Comment:
Maybe I misunderstood.  Don't we `chdir` into the `cl_sandbox` path before we start the tests?  If so, why do we need to test both?  Also, it would be nice to make sure that `GetVolumeInformation` succeeds so that we don't start blindly skipping tests.  (Just wrapping these calls in a `cl_win32_pass` should suffice.)